db-builder/src/main/java/com/yahoo/covid19/database/DatabaseBuilder.java

```diff
@@ -70,6 +70,7 @@
     private static final String CREATE_PLACE = "CREATE TABLE IF NOT EXISTS place\n" +
             "(\n" +
             "    id VARCHAR(255) DEFAULT NOT NULL,\n" +
+            "    type VARCHAR(255) DEFAULT NULL,\n" +
```

Comment:
We'll probably need to make this a list of types - check out JsonType in Elide code to see how to turn a string into a JSON map in the Elide models.